cql3/functions/functions.cc

```diff
@@ -117,15 +117,15 @@ functions::init() noexcept {
     return ret;
 }
 
-void functions_changer::add_function(shared_ptr<function> func) {
+void functions::add_function(shared_ptr<function> func) {
     if (find(func->name(), func->arg_types())) {
```

Comment:
> This reverts commit https://github.com/scylladb/scylladb/commit/3f1c2fecc2ace0604bf742ae8d2f2e9e4a682b4b.

Need to explain why. Or every commit it eligible for reverting without explanation.

Reply:
Also, is the code bisectable after revert?

Reply:
yes, it compiles on every commit

Reply:
> Need to explain why

Will add more info in cover letter

Reply:
The commit itself needs to explain why it's reverting.

Reply:
And I'm confused. How does it still work? functions_changer was added for a reason. The reason isn't gone. What's going on?